src/huggingface/configs/config.jl

```diff
@@ -0,0 +1,30 @@
+using JSON
+
+abstract type AbstractHGFConfig <: AbstractDict{Symbol, Any} end
+
+Base.length(cfg::AbstractHGFConfig) = fieldcount(typeof(cfg))
+Base.isempty(::AbstractHGFConfig) = false
+Base.getindex(cfg::AbstractHGFConfig, k::Symbol) = getfield(cfg, k)
+Base.iterate(cfg::AbstractHGFConfig) = iterate(cfg, 1)
+Base.iterate(cfg::AbstractHGFConfig, i) = i <= length(cfg) ? (fieldname(HGFBertConfig, i)=> getfield(cfg, i), i+1) : nothing
```

Comment:
Given that `AbstractHGFConfig <: AbstractDict{Symbol, Any}`, is it possible just stick with the [convention](https://docs.julialang.org/en/v1/base/collections/index.html#Dictionaries-1) that:

> An AbstractDict{K, V} should be an iterator of Pair{K, V}.

>

Reply:
It's already a iterator of `Pair{Symbol, Any}`. If you do `collect(::AbstractHGFConfig)`, you will get a `Array{Pair{Symbol, Any}, 1}`.

Reply:
Ah yes. I overlooked the 9th line.